package.json

```diff
@@ -74,6 +74,7 @@
     "husky": "4.3.8",
     "jest": "26.6.3",
     "lint-staged": "10.5.3",
+    "lodash.get": "^4.4.2",
```

Comment:
Could you set this to a fixed version :) 

I see node-fetch is also not fixed, could also set that one to the latest version?